frontend/src/pages/index.ts

```diff
@@ -1,4 +1,5 @@
 import { Apply } from "./Apply.tsx";
 import { Candidates } from "./Candidates.tsx";
```

Comment:
Same for these warnings a run of `npm run lint-fix` in the approprate directory should work.